### PR TITLE
fix: revert level 4 media query

### PR DESCRIPTION
**Pull request by zhangyouxin**
## Description 

The new syntax of media query is not compatible with old version chrome(103 and earlier) and causing some bugs on page.

ref: https://caniuse.com/css-media-range-syntax

related bugs: 

- https://github.com/Magickbase/ckb-explorer-public-issues/issues/243#issuecomment-1687948088
- https://github.com/Magickbase/ckb-explorer-public-issues/issues/411

### CODE REVIEW

**Comment by Keith-CY** [comment]
Are these changes made manually? I thought they are replaced by `stylelint` in PR https://github.com/Magickbase/ckb-explorer-frontend/pull/26, and could be reverted by adding rule `media-feature-range-notation: prefix'

Ref: https://stylelint.io/user-guide/rules/media-feature-range-notation/#:~:text=Specify%20context%20or%20prefix%20notation%20for%20media%20feature%20ranges.&text=Media%20features%20of%20the%20range,be%20limiting%20in%20certain%20situations.

**Comment by WhiteMinds** [comment]
In theory, this syntax should be compiled into the old syntax. I suspect that the syntax written in `styled.tsx` was not compiled properly. We need to investigate the reason and confirm whether all styles written in `styled` components have not been compiled for compatibility.

**Comment by zhangyouxin** [comment]
https://github.com/Magickbase/ckb-explorer-frontend/assets/3870972/f3481d4e-d4c4-4330-ad88-31a81354efde

**Comment by FrederLu** [comment]
The hash information in the transaction list on the homepage of the mobile terminal exceeds the display area.
image

File: src/pages/Transaction/TransactionCellScript/styled.tsx [hunk 1]
```diff
@@ -37,35 +37,35 @@ export const TransactionDetailLock = styled(TransactionDetailItem)``
...
 export const TransactionDetailCapacityUsage = styled(TransactionDetailItem)`
   margin-left: 90px;
 
-  @media (width <= 750px) {
+  @media (max-width: 750px) {
     margin-left: 12px;
   }
 
-  @media screen and (width >= 751px) and (width <= 1300px) {
+  @media screen and (width >= 751px) and (max-width: 1300px) {
```
Comment (Keith-CY) [inline src/pages/Transaction/TransactionCellScript/styled.tsx:68]: should be updated

File: src/pages/Transaction/TransactionCellScript/styled.tsx [hunk 2]
```diff
@@ -37,35 +37,35 @@ export const TransactionDetailLock = styled(TransactionDetailItem)``
...
 export const TransactionDetailData = styled(TransactionDetailItem)`
   margin-left: 90px;
 
-  @media (width <= 750px) {
+  @media (max-width: 750px) {
     margin-left: 12px;
   }
 
-  @media screen and (width >= 751px) and (width <= 1300px) {
+  @media screen and (width >= 751px) and (max-width: 1300px) {
```
Comment (Keith-CY) [inline src/pages/Transaction/TransactionCellScript/styled.tsx:56]: should be updated

File: src/pages/Transaction/TransactionCellScript/styled.tsx [hunk 3]
```diff
@@ -37,35 +37,35 @@ export const TransactionDetailLock = styled(TransactionDetailItem)``
 export const TransactionDetailType = styled(TransactionDetailItem)`
   margin-left: 90px;
 
-  @media (width <= 750px) {
+  @media (max-width: 750px) {
     margin-left: 12px;
   }
 
-  @media screen and (width >= 751px) and (width <= 1300px) {
+  @media screen and (width >= 751px) and (max-width: 1300px) {
```
Comment (Keith-CY) [inline src/pages/Transaction/TransactionCellScript/styled.tsx:44]: should be updated

**Comment by zhangyouxin** [comment]
> The hash information in the transaction list on the homepage of the mobile terminal exceeds the display area. image

updated

**Review by Keith-CY — APPROVED** [review]

**Review by FrederLu — APPROVED** [review]

**Pull request merged**